odl/tomo/backends/astra_setup.py

```diff
@@ -161,26 +170,29 @@ def astra_volume_geometry(reco_space):
     NotImplementedError
         If the cell sizes are not the same in each dimension.
     """
-    if not isinstance(reco_space, DiscreteLp):
-        raise TypeError('`reco_space` {!r} is not a DiscreteLp instance'
-                        ''.format(reco_space))
+    if not isinstance(vol_space, DiscreteLp):
+        raise TypeError('`vol_space` {!r} is not a DiscreteLp instance'
+                        ''.format(vol_space))
 
-    if not reco_space.is_uniform:
-        raise ValueError('`reco_space` {} is not uniformly discretized')
+    if not vol_space.is_uniform:
+        raise ValueError('`vol_space` {} is not uniformly discretized')
 
-    vol_shp = reco_space.partition.shape
-    vol_min = reco_space.partition.min_pt
-    vol_max = reco_space.partition.max_pt
+    vol_shp = vol_space.partition.shape
+    vol_min = vol_space.partition.min_pt
+    vol_max = vol_space.partition.max_pt
 
-    if reco_space.ndim == 2:
+    if vol_space.ndim == 2:
         # ASTRA does in principle support custom minimum and maximum
         # values for the volume extent also in earlier versions, but running
         # the algorithm fails if voxels are non-isotropic.
-        if (not reco_space.partition.has_isotropic_cells and
-                not astra_supports('anisotropic_voxels_2d')):
+        if (
+            not vol_space.partition.has_isotropic_cells
+            and not astra_supports('anisotropic_voxels_2d')
+        ):
             raise NotImplementedError(
                 'non-isotropic pixels in 2d volumes not supported by ASTRA '
-                'v{}'.format(ASTRA_VERSION))
+                'v{}'.format(ASTRA_VERSION)
```

Comment:
Perhaps clarify from what version this is supported? E.g. v < x.y.z?

Reply:
Good suggestion, requires a new function though.